csharp/device/Microsoft.Azure.Devices.Client/IotHubConnection.cs

```diff
@@ -305,7 +319,12 @@ AmqpSettings CreateAmqpSettings()
         static AmqpLinkSettings SetLinkSettingsCommonProperties(AmqpLinkSettings linkSettings, TimeSpan timeSpan)
         {
             linkSettings.AddProperty(IotHubAmqpProperty.TimeoutName, timeSpan.TotalMilliseconds);
+#if WINDOWS_UWP
+            // System.Reflection.Assembly.GetExecutingAssembly() does not exist for UWP, therefore use a hard-coded version name
+            linkSettings.AddProperty(IotHubAmqpProperty.ClientVersion, "1.0-UWP");
```

Comment:
You should put this in a constant for better visibility. Also the value should be automatically updated by our bump_release script (the way we do now for other libraries).